chromium_src/chrome/app/chrome_main_delegate.h

```diff
@@ -6,9 +6,8 @@
 #ifndef BRAVE_CHROMIUM_SRC_CHROME_APP_CHROME_MAIN_DELEGATE_H_
 #define BRAVE_CHROMIUM_SRC_CHROME_APP_CHROME_MAIN_DELEGATE_H_
 
-#include "../../../../chrome/common/chrome_content_client.h"
-
 #include "brave/common/brave_content_client.h"
+#include "chrome/common/chrome_content_client.h"
```

Comment:
why was it wrong before and correct now?..

Reply:
relative include paths should only be used for including the original file so this could have caused problems if we have an override for chrome_content_client.h